src/Database/Type/BinaryType.php

```diff
@@ -55,6 +55,9 @@ public function toPHP($value, Driver $driver)
         if ($value === null) {
             return null;
         }
+        if (is_string($value) && preg_match('/^[a-zA-Z0-9]+$/', $value)) {
```

Comment:
I think I would prefer checking explicitly if driver is SQLServer rather than running a preg on a potential couple hundred megabyte strings.